### ISSUE TITLE
 module 'pygments' has no attribute 'formatters'

### ISSUE THREAD

**Issue by bmoniey** [SojiroFukuda/sview-gui#5]
Windows 10
Using virtual env with python 3.8.3
Installed using python -m setup.py install
From python console ran the example below

Issue with pygments.formatters.HtmlFormatter

Python 3.8.3 (tags/v3.8.3:6f8c832, May 13 2020, 22:20:19) [MSC v.1925 32 bit (Intel)] on win32
>>> import sviewgui.sview as sv
... sv.buildGUI()
Backend Qt5Agg is interactive backend. Turning interactive mode on.
Traceback (most recent call last):
  File "<input>", line 2, in <module>
  File "C:\Users\MORANB\PycharmProjects\sview-gui-master\venv\lib\site-packages\sviewgui-0.3.5-py3.8.egg\sviewgui\sview.py", line 1760, in buildGUI
    wmain = Csviwer()
  File "C:\Users\MORANB\PycharmProjects\sview-gui-master\venv\lib\site-packages\sviewgui-0.3.5-py3.8.egg\sviewgui\sview.py", line 100, in __init__
    self.initUI(data)
  File "C:\Users\MORANB\PycharmProjects\sview-gui-master\venv\lib\site-packages\sviewgui-0.3.5-py3.8.egg\sviewgui\sview.py", line 106, in initUI
    self.ui.logbox.document().setDefaultStyleSheet(pygments.formatters.HtmlFormatter(style=Csviwer.STYLE).get_style_defs('.highlight'))
AttributeError: module 'pygments' has no attribute 'formatters'

**Comment by benhiginbotham** [comment]
Apologies if I don't answer in the correct format but I did the following and it worked, using windows 10 and python 3.8.3:

```
import sviewgui.sview as sv
from pygments import formatters, lexers

sv.buildGUI
```
Hope this helps.

**Comment by benhiginbotham** [comment]
Apologies if I don't answer in the correct format but I'm new to Git and Github.
I did the following and it worked, using windows 10 and python 3.8.3:

```
import sviewgui.sview as sv
from pygments import formatters, lexers

sv.buildGUI()
```
Hope this helps.

**Comment by lingyanmeng** [comment]
I met the same issue with python 3.9, and adding `from pygments import formatters, lexers` solved it. I would suggest changing the example and readme, thanks.

**Comment by jyonirou** [comment]
Hello guys, my apologies for being a super late response.
Hopefully, the module error is already fixed in version 1.0.1. I directly added the import line for both lexer and formatted from pygments, which is expected to fix the raised issue.
